executor/calibrate_resource.go

```diff
@@ -196,6 +196,11 @@ func (e *calibrateResourceExec) Next(ctx context.Context, req *chunk.Chunk) erro
 	return e.staticCalibrate(ctx, req, exec)
 }
 
+var (
+	errTooFewMetricsPoints = errors.Normalize("There are too few metrics points available in selected time window, %v")
```

Comment:
Shall we also return the error message `The workload in selected time window is too low ...`  because the user may not know the meaning of `too few metrics points`